### ISSUE TITLE
Plan detailed route : London - Guildford

### ISSUE THREAD

**Issue by grd349** [grd349/CycleTrip#7]
We need to plan the route we are going to take.  I would like to have it in the GPS so that'll be something we need to take care of.  Redundancy in nav would be a good thing too - Paul do you use your phone?

There's no rush on this but I'll start looking into a route.  @pxr329, which train station in London will you come into?

**Comment by pxr329** [comment]
I use a Wahoo Elemnt Bolt cycle computer, it has built in GPS amongst other things, I do pair it to my phone, but that's only to provide live tracking links to friends, or to auto upload to Strava, etc. I plan my routes with Ridewithgps (it's what I planned the original route on if you remember I shared that previously).

I've actually already taken the liberty of creating some routes on there based on the google maps routes you shared for the 5 stages. I can share those if you like. I haven't gone over them with a fine tooth comb yet they are a bit rough. Was going to get them polished once I can put the exact start/end locations after the accommodation is locked in.   

Paddington. What about you? is it Euston or Kings Cross?

**Comment by grd349** [comment]
I will arrive at probably Marylebone station (or Euston but less likely). 

The route the Google gives is always a bit hit and miss.  It doesn't seem to get the Avenue Vert right so I think we should do the fine tooth comb thing.

On the UK google is better but we should take care with central London. :)

**Comment by pxr329** [comment]
Ah I totally forgot about Marylebone station, that's right next to Paddington so will be easy if that's the case. I guess we're better off starting somewhere near those two as we won't need to go over to St Pancras any more. 

Well now we've got a good idea of the start/end points for the stages I'll knock something up in the next day or two and I'll post them. You can take a look at it then and we can tweak as necessary. Ridewithgps lets you export the routes to various different GPS device formats, so you should be able to load it onto what ever you have. They also have a free app, so you could always use that too.

**Comment by pxr329** [comment]
So I've made a start. I've made the routes public so you are able to view them without making an account. I used Strava Heatmaps to plot the route out of the centre, as I figure the popular routes are probably the best to take, they are probably the main commuter routes. Though if you fancy detouring past the odd landmark we could work that in. 

I've got 2 options for you to look at, one is probably best in the dry, could be a bit more picturesque as it follows the Thames and River Wey, but it looks like much of that section is unsealed trail, so could be pretty ropey if the ground is wet. The second alternate route is on road so should be easy no matter the conditions, but goes through some slightly more built up areas. 

1. https://ridewithgps.com/routes/30555659
2. https://ridewithgps.com/routes/30582047

**Comment by grd349** [comment]
Here is the route we followed last year.  I reckon we can take your route and then just tighten it up around Woking and into Guildford.

https://www.strava.com/activities/1797010554

**Comment by pxr329** [comment]
Ah yeah, that seems to follow the river route from the 1st option I suggested. Originally the auto route planner did create the route going through Woking, but I put in a detour as the town centre looked a bit busy, so was just playing it safe. 

I've just edited it to take the Woking detour out, take another look at it now and see what you think.

**Comment by grd349** [comment]
We should try and use this as part of the route ...

https://www.sustrans.org.uk/find-a-route-on-the-national-cycle-network/hampton-court-to-putney/

**Comment by grd349** [comment]
OK - suggestions for the route.  Start Paddington. Go through Hyde Park and head for Pimlico.

Pick up the NCN4 going west.  Follow this all the way to Chertsey then take the NCN 223 all the way to Guildford.  There will be lots of off road cycling which will be nice (if a little slow).  It's Sept so fingers crossed the track will not be too muddy.  

This looks a lot like your route except longer because we stick to the NCN more.

**Comment by pxr329** [comment]
That sounds like a good idea, looks like we've already got the Richmond Park section in there, but I'll add the rest in. I'm heading home in a few minutes so I'll update it later this evening.

**Comment by pxr329** [comment]
If we're heading Pimlico way, how about we divert slightly over to Westminster via St James' Park? or is that a little too much of a divert?

**Comment by pxr329** [comment]
OK I've updated the route to incorporate all of the HC to Putney bits and NCN4 to NCN223 all the way to Guildford. 

The 223 transitions into the 22 in Guildford so I've followed that round and then cut up to the hotel for that final bit. 

I've also done that divert at the start to get a few more POIs in, let me know what you think.

Strange how there is no route from the NCN4 to the NCN221, guess it must have been some bad paths in between. 

Here is the link again so you don't need to keep scrolling up :-)
https://ridewithgps.com/routes/30555659
